src/core/filters/simplefilternode.js

```diff
@@ -60,4 +66,11 @@ export default class SimpleFilterNode extends FilterNode {
   getSimpleAncestors () {
     return this;
   }
+
+  /**
+   * Removes this filter node from the FilterRegistry.
+   */
+  remove () {
+    this._remove();
```

Comment:
It seems like at least a couple of the remove methods don't actually remove the filter from persistent or globalStorage. Instead they set the filter to some empty value in storage. Can we always ensure we're deleting the filter and its remnants?

Reply:
updated!